darwin/future/meta/objects/base.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import Generic, List, Optional, TypeVar
+
+from darwin.future.pydantic_base import DefaultDarwin
+
+R = TypeVar("R", bound=DefaultDarwin)
+
+class MetaBase(Generic[R]):
```

Comment:
I was 50/50 whether to put this, but "metabase" is the name of a popular BI/Dashboarding tool.  I wouldn't normally care, but we do actually use it elsewhere in the stack, so this _might_ be worthy of renaming.

Reply:
The Meta Base object is definitely required for the generics, but don't mind what it's called.